config/services.yml

```diff
@@ -22,40 +22,7 @@ x-turbo-config: &default-turbo-config
       FALLBACK_AUTHN_ENABLED: !Env ${FALLBACK_AUTHN_ENABLED}
 
 https://cmr.earthdata.nasa.gov:
-  - name: nasa/harmony-gdal-adapter
```

Comment:
It looks like this PR removes HGA from production. I was just expecting this entry to shift down to the bottom of the production entries, in the same way the UAT HGA service entry has.

Reply:
The diff came out very confusing for some reason. If you look at the file itself you can see it is still in both places.
OPS: https://github.com/nasa/harmony/blob/e826ee27688fc658d487fc350033e9010d01cf0b/config/services.yml#L412
UAT: https://github.com/nasa/harmony/blob/e826ee27688fc658d487fc350033e9010d01cf0b/config/services.yml#L870

Reply:
I added comments above the entries to make the diff a little bit more clear

Reply:
Ah, yeah. That diff was misleading. Looks good to me, then!